sourcegraph/sourcegraph
# SG-52178 - Add not logged in pages and banners to increase conversion

## Description
Add Cody CTA to the blob and repo page when user is not logged in.

[Figma Design](https://www.figma.com/file/75bhaZyepWto4l1gFUPmQe/Cody's-no-access-pages-for-logged-out-users?type=design&node-id=29-4517&t=nmsBD8ECsB9LgLnE-4)

## Ref
[SG Issue](https://github.com/sourcegraph/sourcegraph/issues/52178)

[Gitstart Ticket](https://clients.gitstart.com/sourcegraph/1/tickets/SG-52178)

## Demo
![Screenshot 2023-06-02 at 11 57 37 AM](https://github.com/sourcegraph/sourcegraph/assets/89894075/182fc5e9-ef89-4a13-945c-20bdb20ebebe)

## Test Plan
1. Run application on this branch.
2. Go to the blob/repo pages as unauthenticated user
3. See that the mockup specified in attached attached Figma design is displayed.
